src/mkvgen/include/com/amazonaws/kinesis/video/mkvgen/Include.h

```diff
@@ -150,6 +150,7 @@ extern "C" {
 #define MKV_AVC_CONTENT_TYPE                ((PCHAR) "video/avc")
 #define MKV_HEVC_CONTENT_TYPE               ((PCHAR) "video/hevc")
 #define MKV_H264_AAC_MULTI_CONTENT_TYPE     ((PCHAR) "video/h264,audio/aac")
+#define MKV_H264_MULAW_MULTI_CONTENT_TYPE     ((PCHAR) "video/h264,audio/mulaw")

```

Comment:
tab alignment

Reply:
@MushMal gladly, I haven't seen tab settings documented anywhere. Would you mind telling me the project's tab settings so I can update both branches?

Reply:
For some of the projects we started enforcing clang format rules (WebRtc SDK). We will be doing it here too at some stage. Currently, we use some defaults which have been more organically established but soon will be the same as defined in WebRtc project.

In this case, the only ask is to align ((PCHAR) "video/h264,audio/mulaw") to the above line

Reply:
@dwaynehubbard - if we can get the alignment to match the rest we should be able to push this through quickly!